python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -153,39 +121,45 @@ def uniform_neighbor_sample(input_graph,
         raise TypeError("fanout_vals must be a list, "
                         f"got: {type(fanout_vals)}")
 
-    ddf = input_graph.edgelist.edgelist_df
-    src_col_name = input_graph.renumber_map.renumbered_src_col_name
-    dst_col_name = input_graph.renumber_map.renumbered_dst_col_name
-
-    weight_t = ddf["value"].dtype
-    if weight_t == "int32":
-        ddf = ddf.astype({"value": "float32"})
-    elif weight_t == "int64":
-        ddf = ddf.astype({"value": "float64"})
-
-    num_edges = len(ddf)
-    data = get_distributed_data(ddf)
+    weight_t = input_graph.edgelist.edgelist_df["value"].dtype
 
     # start_list uses "external" vertex IDs, but if the graph has been
     # renumbered, the start vertex IDs must also be renumbered.
     if input_graph.renumbered:
         start_list = input_graph.lookup_internal_vertex_id(
             start_list).compute()
-    # FIXME: should we add this parameter as an option?
-    do_expensive_check = False
-
-    result = [client.submit(call_nbr_sampling,
-                            Comms.get_session_id(),
-                            wf[1],
-                            src_col_name,
-                            dst_col_name,
-                            num_edges,
-                            do_expensive_check,
-                            start_list,
-                            fanout_vals,
-                            with_replacement,
-                            workers=[wf[0]])
-              for idx, wf in enumerate(data.worker_to_parts.items())]
+
+    '''
+    start_list = dask_cudf.from_cudf(
+        start_list,
+        npartitions=input_graph.npartitions
+    )
+    start_list = get_distributed_data(start_list)
+    wait(start_list)
+    '''
```

Comment:
The part you commented is something we have to address in the future .
I had a discussion with Chuck regarding distributing the seeds(for performance reason) like `BFS` because our current implementation send a copy of the seeds to each rank which results in `n` sampling for `n` seeds:

Reply:
Right, I know this is something we're working on.  I will make a note of that here.